sipa/static/css/agdsn-bootstrap-social.css

```diff
@@ -0,0 +1,67 @@
+.btn-social-icon {
+    position: relative;
+    padding-left: 44px;
+    text-align: left;
+    write-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    height: 34px;
+    width: 34px;
+    padding: 0;
+
+    background-color: var(--btn-social-bg-color);
+    color: #ffffff;
+
+    border: none;
+}
+.btn-social-icon>:first-child{
+    position: absolute;
+    left: 0;
+    top: 0;
+    bottom: 0;
+    width: 32px;
+    line-height: 34px;
+    font-size: 1.6em;
+    text-align: center;
+    border-right: 1px solid rgba(0, 0, 0, 0.2)
+    border: none;
```

Comment:
It's unclear what you want to achieve here.
First, it's missing a semicolon; so this rule is not applied due to the invalid value `1px solid rgba(0, 0, 0, 0.2) border: none`.
Second, when adding the semicolon, you would have the `border: none` overriding the border rule completely.

Reply:
I think I just copy-pasted most of the CSS back then (should've noted that here though).